File: Basic Webcam/Form1.vb
Public Class Form1
    Public Touchless As New TouchlessLib.TouchlessMgr
    Public Camera1 As TouchlessLib.camera = Touchless.Cameras.Item(0)
    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Touchless.CurrentCamera = Camera1
        Touchless.CurrentCamera.CaptureHeight = 480
        Touchless.CurrentCamera.CaptureWidth = 640
        Timer1.Start()
    End Sub
    Private Sub Timer1_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Timer1.Tick
        Dim originalImage As Image = Touchless.CurrentCamera.GetCurrentImage()
        If originalImage IsNot Nothing Then
            Dim mirroredBitmap As New Bitmap(originalImage.Width, originalImage.Height)
            Dim g As Graphics = Graphics.FromImage(mirroredBitmap)
            g.ScaleTransform(-1, 1)
            g.TranslateTransform(-originalImage.Width, 0)
            g.DrawImage(originalImage, New Point(0, 0))
            PictureBox1.Image = mirroredBitmap
            g.Dispose()
        End If
    End Sub
    Private Sub AboutToolStripMenuItem1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles AboutToolStripMenuItem.Click
        MsgBox("Basic Webcam 1.1 - Coded By: Emir Alp Koçak - emiralpkocak@gmail.com", MsgBoxStyle.DefaultButton1, "About")
    End Sub
    Private Sub ExitToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ExitToolStripMenuItem.Click
        Me.Close()
    End Sub

    Private Sub TakePhotoToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles TakePhotoToolStripMenuItem.Click
        If SaveFileDialog1.ShowDialog = Windows.Forms.DialogResult.OK Then
            PictureBox1.Image.Save(SaveFileDialog1.FileName)
        End If
    End Sub
    Private Sub EnableAlwaysOnTopToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles EnableAlwaysOnTopToolStripMenuItem.Click
        If EnableAlwaysOnTopToolStripMenuItem.Checked = False Then
            Me.TopMost = False
        Else
            Me.TopMost = True
        End If
    End Sub

    Private Sub EnableSmallerViewToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles EnableSmallerViewToolStripMenuItem.Click
        If EnableSmallerViewToolStripMenuItem.Checked = False Then
            Me.FormBorderStyle = Windows.Forms.FormBorderStyle.Sizable
        Else
            Me.FormBorderStyle = Windows.Forms.FormBorderStyle.SizableToolWindow
        End If
    End Sub
End Class

File: Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using AForge.Video;
using AForge.Video.DirectShow;

namespace Basic_Webcam
{
    public partial class Form1 : Form
    {
        public FilterInfoCollection cameras;
        private VideoCaptureDevice captureDevice;
        public Form1()
        {
            InitializeComponent();
            this.FormClosing += new FormClosingEventHandler(Form1_FormClosing);
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            try
            {
                cameras = new FilterInfoCollection(FilterCategory.VideoInputDevice);
                foreach(FilterInfo camera in cameras)
                {
                    comboBox1.Items.Add(camera);
                }
                comboBox1.SelectedIndex = 0;
            }
            catch (Exception ex)
            {
                MessageBox.Show("Cannot find a camera device. Please check if your camera is connected and required drivers are installed. Error: " + ex.Message, "Error", MessageBoxButtons.OK, MessageBoxIcon.Exclamation);
            }
        }
        private void source_content(object sender, NewFrameEventArgs e)
        {
            Bitmap content = (Bitmap)e.Frame.Clone();
            Bitmap mirrored = new Bitmap(content.Width, content.Height);
            using (Graphics graphic = Graphics.FromImage(mirrored))
            {
                graphic.TranslateTransform(content.Width, 0);
                graphic.ScaleTransform(-1, 1);
                graphic.DrawImage(content, 0, 0);
            }
            pictureBox1.Image = mirrored;
            content.Dispose();
        }
        private void Form1_FormClosing(object sender, EventArgs e)
        {
            try
            {
                if (captureDevice.IsRunning)
                {
                    captureDevice.Stop();
                    Application.Exit();
                    Environment.Exit(0);
                }
                Application.Exit();
                Environment.Exit(0);
            }
            catch (Exception ex)
            {
                Application.Exit();
                Environment.Exit(0);
            }
        }

        private void takePhotoToolStripMenuItem_Click(object sender, EventArgs e)
        {
            if (saveFileDialog1.ShowDialog() == DialogResult.OK)
            {
                try
                {
                    pictureBox1.Image.Save(saveFileDialog1.FileName, System.Drawing.Imaging.ImageFormat.Png);
                    MessageBox.Show("Photo saved to: " + saveFileDialog1.FileName);
                }
                catch (Exception ex)
                {
                    MessageBox.Show("Unable to save picture. Error: " + ex.Message, "Error", MessageBoxButtons.OK, MessageBoxIcon.Warning);
                }
            }
        }

        private void enableAlwaysOnTopToolStripMenuItem_Click(object sender, EventArgs e)
        {
            if (enableAlwaysOnTopToolStripMenuItem.Checked)
            {
                this.TopMost = true;
            }
            else
            {
                this.TopMost = false;
            }
        }

        private void enableCompactViewToolStripMenuItem_Click(object sender, EventArgs e)
        {
            if (enableCompactViewToolStripMenuItem.Checked)
            {
                this.FormBorderStyle = FormBorderStyle.SizableToolWindow;
            }
            else
            {
                this.FormBorderStyle = FormBorderStyle.Sizable;
            }
        }

        private void selectCameraToolStripMenuItem_Click(object sender, EventArgs e)
        {
            if (captureDevice.IsRunning)
            {
                captureDevice.Stop();
            }
            pictureBox1.Image = null;
            label1.Show();
            comboBox1.Show();
            button1.Show();
            comboBox1.Items.Clear();
            try
            {
                cameras = new FilterInfoCollection(FilterCategory.VideoInputDevice);
                foreach (FilterInfo camera in cameras)
                {
                    comboBox1.Items.Add(camera);
                }
                comboBox1.SelectedIndex = 0;
            }
            catch (Exception ex)
            {
                MessageBox.Show("Cannot find a camera device. Please check if your camera is connected and required drivers are installed. Error: " + ex.Message, "Error", MessageBoxButtons.OK, MessageBoxIcon.Exclamation);
            }
        }

        private void exitToolStripMenuItem_Click(object sender, EventArgs e)
        {
            try
            {
                if (captureDevice.IsRunning)
                {
                    captureDevice.Stop();
                    Application.Exit();
                    Environment.Exit(0);
                }
                Application.Exit();
                Environment.Exit(0);
            }
            catch (Exception ex)
            {
                Application.Exit();
                Environment.Exit(0);
            }
        }

        private void startStopCameraToolStripMenuItem_Click(object sender, EventArgs e)
        {
            try
            {
                if (captureDevice.IsRunning)
                {
                    captureDevice.Stop();
                }
                else
                {
                    captureDevice.Start();
                }
            }
            catch (Exception ex)
            {
                MessageBox.Show("Cannot access camera device. Error: " + ex.Message, "Error", MessageBoxButtons.OK, MessageBoxIcon.Exclamation);
            }
        }

        private void aboutToolStripMenuItem_Click(object sender, EventArgs e)
        {
            MessageBox.Show("Basic Webcam v1.2.0 - Coded By: Emir Alp Koçak", "About");
        }

        private void button1_Click(object sender, EventArgs e)
        {
            try
            {
                captureDevice = new VideoCaptureDevice();
                captureDevice = new VideoCaptureDevice(cameras[comboBox1.SelectedIndex].MonikerString);
                captureDevice.NewFrame += new NewFrameEventHandler(source_content);
                captureDevice.Start();
                comboBox1.Hide();
                label1.Hide();
                button1.Hide();
            }
            catch (Exception ex)
            {
                MessageBox.Show("Cannot access camera device. Error: " + ex.Message, "Error", MessageBoxButtons.OK, MessageBoxIcon.Exclamation);
            }
        }
    }
}
